### ISSUE TITLE
Change "paths that may treat conditions related to" phrasing under each result in the UI

### ISSUE THREAD

**Issue by mbrush** [NCATSTranslator/Feedback#793]
e.g. 
![image](https://github.com/NCATSTranslator/Feedback/assets/5184212/f73af0cd-0ac6-4a7f-98c3-1c5a8db0f68c)

I have two issues with these phrases:
1. My biggest concern is that "Paths" do not treat things.  This makes no sense. Can we be more precise/clear here?
2. Less concerning but worth asking is why we added "conditions related to" . . . aren't our reasoners specifically tuned to find things that may treat the queried condition?  They are not tuned to predict anything about related conditions? And what is even meant by "related to" here?  I suspect this was added as a CYA hedge, but IMO it is inaccurate and confusing. I think there is plenty of language/disclaimers to tell users to take these results as hypotheses to further explore, not treatments to apply to patients. 

------

IMO, these phrases should say something like::
![image](https://github.com/NCATSTranslator/Feedback/assets/5184212/3a996342-e1bd-44be-9601-1058fe7a0080)

---------

If we insist on hedging / expanding the world of possible targets for this treatment, I might say it like:
![image](https://github.com/NCATSTranslator/Feedback/assets/5184212/3c816c5f-96e3-43ee-b6d0-cbcb843a1a25)

**Comment by gglusman** [comment]
Perhaps also change 'Paths' to 'lines of evidence' or somesuch?

**Comment by mbrush** [comment]
As much as I personally like to think of things in terms of "evidence lines", the guidance and labels we use in the UI seems to train our users to think in terms of "Paths".  So I would be hesitant to introduce a new term that is not consistent with other labels / guidance we provide  (unless we update other features of the UI in parallel).

**Comment by sstemann** [comment]
if we change the text in the response, do we also change the text in the question?

**Comment by dnsmith124** [comment]
I agree with @sstemann, the question text and path text should use the same language, so if we change one we should change them both. 

I believe the "may treat conditions related to X" language was implemented as both a CYA measure and a recognition of the fact that ARAs very often return diseases that are not the disease that was directly searched for, most often in the form of subclasses of the searched for disease. The original "# paths that may treat X" was confusing to users because oftentimes many paths don't show a potential treatment for X. 

@reeserosedesign @Genomewide It's probably worth user testing to see if the text after the number of paths is even necessary, since the user should be aware of what it was they searched for (and the current query is stated at the top of the results page). Perhaps we can sidestep this problem altogether.

**Comment by mbrush** [comment]
Thanks for walking through this all.  I get that the "or related conditions" is a CYA measure, and covers cases where the condition that comes back for a result is a subtype of the queried condition.  I would still argue that "related conditions" is too broad however.   If conditions that come back with a Result is always the queried condition or a subtype of it - the text should reflect this:

"14 paths support this possible treatment for Cerebral Palsy or subtypes of this condition"

That said, my **_primary concern in this ticket_** was less about the _"paths that treat"_ text - as this is not meaningful or accurate.   "Paths" don't treat conditions.  Regardless of how we handle the "related conditions" issue, can we at least change _"paths that treat"_ to something like _"paths support this possible treatment for"_?

Also fine with me if we simplify further, as proposed above, and remove the text after the number of paths altogether.  e.g. just say "14 paths"  or "14 support paths" under a given Result chemical.

**Comment by sstemann** [comment]
@dnsmith124 i think going to user testing is a good step.

**Comment by dnsmith124** [comment]
Completely understand your concerns @mbrush, we'll come up with some options for different language here and test with users 👍

**Comment by gglusman** [comment]
"14 support paths" is a really good option: simple, generic (no need to tailor it for different query types), minimizes further hair-splitting ("why do you call this well-established drug a 'possible' treatment??"), etc.

If including the subject name and object name is important, perhaps: "14 paths connecting <subject> to <object>", or "14 paths supporting this relationship between <subject> and <object>".
